tests/stress/buildpreprocess.py

```diff
@@ -0,0 +1,79 @@
+import csv
+import sys
+from pathlib import Path
+
+# this program will generate a csv file with a predefined number of colums and rows (preferabli each value unique)
+
+
+def generate_maximally_complex_csv(
+    numb_rows: int, temp_dir: Path = Path("temp_dir"), max_num_measures: int = 20
+):
+
+    temp_dir.mkdir(exist_ok=True)
```

Comment:
Could you add a comment to make it clear that the output of this function is to be paired with `test-qube-config.json` to ensure we correctly identify the type of each column when we build the CSV-W?